pkg/exporter/sumologicexporter/exporter_test.go

```diff
@@ -796,3 +808,72 @@ func TestSendEmptyTraces(t *testing.T) {
 	err := test.exp.pushTracesData(context.Background(), traces)
 	assert.NoError(t, err)
 }
+
+func TestGetSignalUrl(t *testing.T) {
+	testCases := []struct {
+		description  string
+		signalType   component.Type
+		cfg          *Config
+		endpointUrl  string
+		expected     string
+		errorMessage string
+	}{
+		{
+			description: "no change if log format not otlp",
+			signalType:  component.DataTypeLogs,
+			cfg:         &Config{LogFormat: TextFormat},
+			endpointUrl: "http://localhost",
+			expected:    "http://localhost",
+		},
+		{
+			description: "no change if metric format not otlp",
+			signalType:  component.DataTypeMetrics,
+			cfg:         &Config{MetricFormat: PrometheusFormat},
```

Comment:
should we have one test for otlp format, if I understand that would change the Endpoint to
`http://localhost/v1/metrics`

Reply:
I could add similar tests for logs and metrics. I originally thought they were redundant, but maybe that's fine if it makes the behaviour clearer.